niworkflows/tests/test_viz.py

```diff
@@ -75,3 +78,9 @@ def test_plot_melodic_components():
     viz.utils.plot_melodic_components(melodic_dir, 'in_file.nii.gz', tr=2.0,
                                       report_mask='report_mask.nii.gz',
```

Comment:
post-hoc review, I see I wrote this test with incorrect assumptions, the `noise_components_file` [should exist](https://github.com/maartenmennes/ICA-AROMA/blob/b53bd93c6cad34655d5bc122168351eeb6652e1c/ICA_AROMA_functions.py#L501-L506), but be empty if there were no noise components identified.

I noticed this because both [no_noise](https://605-74174878-gh.circle-artifacts.com/0/tmp/tests/melodic_no_noise.svg) has no warning printed (but should), but [no_classified](https://605-74174878-gh.circle-artifacts.com/0/tmp/tests/melodic_no_classified.svg) is correctly printed without a warning.

I can open a quick pull request to rectify this mistake since you didn't touch this code.

Reply:
Oops, saw that you already caught and fixed that mistake with [this commit](https://github.com/poldracklab/niworkflows/commit/db69566419afb9b9152b5f15ae5ed0f9601b4b6d). sorry for the noise.